src/sections/node/tickets.tsx

```diff
@@ -44,28 +44,12 @@ function TicketsPage() {
   };
 
   const handleRedeemAllTickets = () => {
-    set_redeeming(true);
     dispatch(
       actionsAsync.redeemTicketsThunk({
         apiEndpoint: loginData.apiEndpoint!,
         apiToken: loginData.apiToken!,
       }),
-    )
-      .unwrap()
-      .then(() => {
-        set_redeeming(false);
-        handleRefresh();
```

Comment:
noticed that handleRefresh is no longer going to be called. Maybe not needed let me know.

Reply:
It is necessary, nice catch. I will move the handle refresh to happen after the reeemTickets